### DISCUSSION TITLE
Recent created domain fails DNS checks

### DISCUSSION THREAD

**Discussion by lpossamai** [modoboa/modoboa#2244] [Q&A] [answered]
# Impacted versions

* OS Type: Ubuntu
* OS Version: 20.94.2 LTS
* Database Type: PostgreSQL
* Database version: 12.6
* Modoboa: 1.17
* installer used: Yes
* Webserver: Nginx

# Steps to reproduce

Add a new domain via the Web UI, with no DNS records yet created.

You should get the message below from Modoboa:
![image](https://user-images.githubusercontent.com/17607576/115656552-827cdc00-a389-11eb-9fec-262f958fc90a.png)

# Current behavior

After adding a new domain with no (yet) DNS records created, I keep getting the `No DNS record found` error message, and my domain is stuck at `Awaiting checks` status.

# Expected behavior

Modoboa should have a way to trigger the DNS checks again. There is no way to manually check the DNS again after adding the domain.

Issue https://github.com/modoboa/modoboa/issues/1023 would fix this.

**Comment by lpossamai** [comment, marked as answer]
After adding the DNS records required by Modoboa and running `/srv/modoboa/env/bin/python /srv/modoboa/instance/manage.py modo check_mx` manually, I was able to get the domain working.

So back to #1023 where I think this should be in the Admin Web UI.

**Reply by tonioo** [reply]
When you create a domain, there is an option to disable/enable DNS checks. You could have disable them at creation and enable them once your DNS record was created.

**Reply by lpossamai** [reply]
Hmm, I couldn't find that option
